### DISCUSSION TITLE
Github Universe Recordings?

### DISCUSSION THREAD

**Discussion by bittin** [community/community#22216] [New to GitHub]
<p>Todays Github Universe Keynote, collided with another Online thing i am attending tonight, is it recorded and uploaded somewhere tomorrow morning?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>There’s a Youtube embed on the talk page already:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/5814919/181069583-d32b2c9e-18f3-4b41-b18e-58e3c542e6e4.png" class="site-icon" width="72" height="72">
      <a href="https://githubuniverse.com/opening-keynote/" target="_blank" rel="noopener" title="12:00AM - 25 September 2020">GitHub Universe · Virtual 2020 · Dec 8 - 10, 2020 – 25 Sep 20</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/360;"><img src="https://user-images.githubusercontent.com/5814919/181069584-95a0736f-d7cb-4c9f-8d67-6ec9676019a7.jpeg" class="thumbnail" width="690" height="360" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/1/71cdd6afaf0f17a369252250630714d7b3b6b417_2_690x360.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/7/1/71cdd6afaf0f17a369252250630714d7b3b6b417_2_1035x540.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/1/71cdd6afaf0f17a369252250630714d7b3b6b417.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/1/71cdd6afaf0f17a369252250630714d7b3b6b417_2_10x10.png"></div>

<h3><a href="https://githubuniverse.com/opening-keynote/" target="_blank" rel="noopener">Opening keynote</a></h3>

<p>Join Nat Friedman and special guests for an inside look at the latest updates and product announcements from GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by bittin** [comment]
<p>Cool thanks, will watch later today<br>
Marking as solved</p>
